src/frontend/context/AppContext.tsx

```diff
@@ -62,7 +64,25 @@ const [AppProvider, useApp] = createUseContext(({ innloggetSaksbehandler }: IPro
             });
     };
 
-    return { axiosRequest, autentisert, åpneModal, lukkModal, modal, settModal };
+    const hentSaksbehandlerRolle = (): BehandlerRolle | undefined => {
```

Comment:
Kunne denne by default returnert veileder? Eventuelt ukjent som kicker deg ut?

Fordi da kunne man satt Ukjent øverst også bare iterert gjennom rollene og hvis man finner noe med høyere rank bytter man ut. Tilslutt returnerer man rolle og den er enten ukjent eller høyere :)

Reply:
Vi kunne egentlig logget feilen til backend (loggFeil()) og sendt brukeren til /error hvis man ikke har tilgang.

Reply:
Fint med logging! Vurderte å default returnere laveste (nå system), slik du sier, men tenker egentlig at man ikke automatisk bør bli tildelt en rolle dersom man ikke tilhører noen av de definerte gruppene.

Reply:
Har oppdatert litt np, men lurer på hva/hvordan jeg sender brukeren til error. Er det `error.html` det er snakk om? :) Bruker man i så fall noe ala
`axiosRequest<void, void>({
            method: 'GET',
            url: '/error',
        });`

eller bør det defineres en funksjon sammen med `loggFeil`?

Reply:
Har gjort et par endringer nå. History fikk jeg ikke brukt på toppnivå i createUseContext, hvor vi ønsker å ha det. Forslag? Eventuelt tenker jeg også det er ok at henting av rolle ikke gjør alt for mye, men mottakeren har ansvar for å bestemme hva som gjøres med undefines (hhv. sende til error-page)

Reply:
Ja, greit det! 👍